packages/edit-site/src/components/block-editor/use-navigate-to-entity-record.js

```diff
@@ -16,7 +16,9 @@ export default function useNavigateToEntityRecord() {
 
 	const onNavigateToEntityRecord = useCallback(
 		( params ) => {
-			history.push( { ...params, focusMode: true, canvas: 'edit' } );
+			history.navigate(
+				`/${ params.postType }/${ params.id }?canvas=edit&focusMode=true`
```

Comment:
Doing the query building feels a bit tedious, especially if we're handling the encoding ourselves. Should we consider using `addQueryArgs()` or similar in all these?

Reply:
I already use addQueryArgs when the query is dynamic (variables), in this particular case, it feels unnecessary to me.